packages/api/src/controllers/access-control.ts

```diff
@@ -138,9 +138,14 @@ app.post(
     if (
       playbackPolicyType !== "public" &&
       req.body.pub === req.config.accessControlAdminPubkey &&
+      req.config.accessControlAdminPubkey !== "" &&
+      req.config.accessControlAdminPubkey &&
```

Comment:
This check already contains the above since `""` is falsy
```
$ node
Welcome to Node.js v21.4.0.
> if ("") {
... console.log("true")
... } else {
... console.log("false")
... }
false
```